<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PPG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GraphicsLayoutWidget" name="FIR_Response">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>470</width>
      <height>510</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>470</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>470</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="Original_Data">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>690</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="Spectrum">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>690</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="Filtered_Data">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>690</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="BPM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>470</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Average Heart Beat: 0.00 bpm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
